kong/plugins/oauth2/schema.lua

```diff
@@ -31,6 +31,7 @@ return {
     accept_http_if_already_terminated = { required = false, type = "boolean", default = false },
     anonymous = {type = "string", default = "", func = check_user},
     global_credentials = {type = "boolean", default = false},
+    refresh_token_ttl = {required = false, type = "number", default = 1209600} -- original hardcoded value - 14 days
```

Comment:
Do not allow `nil` here, so `required = true`.

This also means that a migration is needed to insert the default value in all existing entries, see [this code](https://github.com/Mashape/kong/blob/next/kong/plugins/key-auth/migrations/postgres.lua#L29-L47) for an example.

Because of the migration, we also cannot target the `master` branch, but must go to the next major release, which is in the `next` branch.

Reply:
OK, gotcha, let me add the `required` flag, the migration and rebase/switch to `next` branch.